x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/ilm/LifecycleOperationSnapshotTests.java

```diff
@@ -47,7 +48,7 @@ public class LifecycleOperationSnapshotTests extends ESSingleNodeTestCase {
 
     @Override
     protected Collection<Class<? extends Plugin>> getPlugins() {
-        return List.of(LocalStateCompositeXPackPlugin.class, IndexLifecycle.class);
+        return List.of(LocalStateCompositeXPackPlugin.class, IndexLifecycle.class, SnapshotLifecycle.class);
```

Comment:
Should this test be split? (follow-up PR) 

I think it's testing both ILM and SLM functionality so we could benefit from having 2 tests, each test residing in the module it tests.

Reply:
Isn't it testing the interaction of SLM and ILM? Ideally it would probably be in a 3rd plugin that depends on both, but it seemed like overkill for an integration test.